mantle/cmd/kola/qemuexec.go

```diff
@@ -323,6 +323,10 @@ func runQemuExec(cmd *cobra.Command, args []string) error {
 		}
 		builder.Memory = int(parsedMem)
 	}
+	// SE needs at least 4GB, so adjust if it's less than required
+	if kola.QEMUOptions.SecureExecution && builder.Memory < 4096 {
```

Comment:
I was thinking more something like:

```go
} else if kola.QEMUOptions.SecureExecution {
	builder.Memory = 4096
}
```

?

That way we don't change a value that the developer explicitly passed in via `--memory` (even if it's smaller than 4G), which I think can be surprising.

Reply:
User by mistake can provide less memory than required, so why should we allow this ? i think it's better to adjust memory to 4gb, than let user see failing VM.

Reply:
Yeah, I see what you mean. But I think if the user directly specified a value, we should respect it, even if it seems wrong, and assume they know what they're doing. If they don't, then worst case the VM will fail to boot and they'll fix it. I think overriding explicit CLI arguments [would be surprising](https://en.wikipedia.org/wiki/Principle_of_least_astonishment).